styles/picrew/catppuccin.user.css

```diff
@@ -761,10 +761,13 @@
       border-right-color: @subtext0 !important;
     }
     /* :face_with_spiral_eyes: */
-    .tc_items_itemlist.is_style_detail > li .tc_items_itemlist_content .tc_items_itemlist_layers > li .tc_items_itemlist_imgs > li > div,
-    .tc_items_itemlist.is_style_detail > li .tc_items_itemlist_content .tc_items_itemlist_layers > li .tc_items_itemlist_imgs > li > div::before {
-      background-color: @surface0 !important;
-      border-color: @surface0 !important;
+```suggestion
```

Comment:
oopsies, looks like you commited the word suggestion too

Reply:
yup i just saw that, i'll edit that out in the next commit

Reply:
FYI there is a little button to accept the commit suggestion instead of manually copying it over :)